mobot.mk

```diff
@@ -65,11 +66,10 @@ scrcpy: build-app
 test:
 	rm -rf build_test
 	CFLAGS="-DMOBOT_VERSION='\"$(GIT_DESCRIBE)/$(ARCH)\"'" \
-	meson build_test --buildtype debug --strip -Db_sanitize=address -Db_lto=true -Dlocal_libav=build-libav -Dcompile_server=false
+	meson build_test --buildtype debug --strip -Db_sanitize=address -Db_lto=true -Dlocal_libav=build-libav -Dcompile_server=false -Dusb=false
```

Comment:
Should we also set `-Dcompile_server=true` here? Or is that unnecessary for tests?

Reply:
It's unnecessary for tests.